frontend/src/components/Layout/index.tsx

```diff
@@ -0,0 +1,22 @@
+import clsx from "clsx";
+import { JSX, PropsWithoutRef } from "react";
+import styles from "./Layout.module.css";
+
+type LayoutProps = Overwrite<PropsWithoutRef<JSX.IntrinsicElements["main"]>, {
+  verticallyCenter?: boolean;
+}>
+
+export default function Layout({ children, className, verticallyCenter = false, ...props }: LayoutProps) {
+  return (
+    <main
+      className={clsx(
+        styles.layout,
+        verticallyCenter && styles.verticallyCenter,
+        className,
+      )}
+      {...props}
+    >
+      {children}
+    </main>
```

Comment:
Just to be consistent 💐 

```suggestion
    <main
      {...props}
      className={clsx(
        styles.layout,
        verticallyCenter && styles.verticallyCenter,
        className,
      )}
    >
      {children}
    </main>
```